zig.yaml

```diff
@@ -0,0 +1,63 @@
+package:
+  name: zig
+  version: 0.10.0
+  epoch: 0
+  description: "General-purpose programming language designed for robustness, optimality, and maintainability"
+  target-architecture:
+    - all
+  copyright:
+    - paths:
+      - "*"
+      attestation: TODO
+      license: MIT
+
+environment:
+  contents:
+    packages:
+      - wolfi-base
+      - busybox
+      - cmake
+      - clang-15
+      - clang-15-dev
+      - build-base
+      - libstdc++
+      - llvm15
+      - llvm15-dev
+      - llvm-lld
+      - libxml2-dev
+      - llvm-lld-dev
+      - samurai
+      - zlib-dev
+
+pipeline:
+  - uses: fetch
+    with:
+      uri: https://github.com/ziglang/zig/archive/${{package.version}}.tar.gz
+      expected-sha256: 805a3789776ed2835c41d12139568a406c9827bb8363a70d6f96340a95ffa8f2
+  - uses: patch
+    with:
+      patches: zig-0.10.0-compiler_rt-aarch64-outline-atomics.patch
+  - runs: |
+      cmake -B build -G Ninja \
+        -DCMAKE_BUILD_TYPE=Release \
+        -DCMAKE_INSTALL_PREFIX=/usr \
+        -DCMAKE_PREFIX_PATH=/usr \
+        -DCMAKE_VERBOSE_MAKEFILE=ON \
+        -DZIG_VERSION="${{package.version}}" \
+        -DZIG_SHARED_LLVM=ON
+      cmake --build build
+
+      # XXX: Workaround for missing zig binary in the build dir.
```

Comment:
Should this be a TODO at least?

Reply:
This was copied from the alpine one: https://git.alpinelinux.org/aports/tree/testing/zig/APKBUILD#n48